aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/includes/question_list.html

```diff
@@ -31,6 +31,6 @@ <h3 class="panel-title">{{ qgroup.grouper|default:_("Category") }}</h3>
 </div>
 
 {% if list and title %}
-    {% url 'aldryn_faq:faq-category' object.category.slug as category_url %}
+    {% url 'aldryn_faq:faq-category' object.category.pk object.category.slug as category_url %}
```

Comment:
Would it make sense to use {{ object.category.get_absolute_url }} instead? Seems like it might be easier to maintain.